### PR TITLE
Add pacci panels

### PR DESCRIPTION
**Pull request by Endaris**
Miz is providing these to be added under the current zlib license.

I removed the pdp_ta panels as removing IP was the goal of adding these panels.
The game will try and set these panels as the default rather than randomizing between all present default panels.
I also got rid of the duplicate data in `panels/__default` and `default_data/panels/` as Miz's basic panels had all of their files present twice.

### CODE REVIEW

**Comment by Shosoul** [comment]
I believe they are supposed to be present twice for how it is set up since we need that for panels to get deployed to the user's directory. Probably shouldn't be done like that, but it is what it is

File: panels.lua [hunk 1]
```diff
@@ -64,14 +64,19 @@ function panels_init()
   add_panels_from_dir_rec("panels")
   
   if #panels_ids == 0 or (config and not config.defaultPanelsCopied) then
-    config.defaultPanelsCopied = true
+    recursive_copy("panels/__default", "panels/pacci")
```
Comment (Endaris) [inline panels.lua:67]: I explicitly added a second call to copy to make sure it does get copied on first startup

**Review by Shosoul — APPROVED** [review]
This is good to go, but I will note that people who already have the game will not get pacci panels as an option or have them installed automatically since it isn't present in the panels folder

**Comment by Endaris** [comment]
> This is good to go, but I will note that people who already have the game will not get pacci panels as an option or have them installed automatically since it isn't present in the panels folder

I tested this again and it doesn't matter whether it's `default_data/panels` or `panels/__default`, none of them will get loaded. I put all my panels into a `__hide` directory, started up on this branch, got the new ones copied, then hid both pacci and HD basic away and when I start the game again, I only have the inverted panel attack panels. 
`default_data/panels` was originally the only source for the copying process to `panels` on first startup but would never get loaded separately. `panels/__default` doesn't get loaded either thanks to the `__` prefix but will be used as the fallback in the case of missing assets of another mod. And due to my change it gets copied as well now.

But no matter where I put the files, to bring pacci to existing clients I would need to use a separate solution. And that is where the snake kind of bites its tail because if I do it similar to what we have for themes, we get the same issues as for themes, namely #858 (which could be a thing if colorblind users want to adjust singular colors) and the fact that it becomes impossible to remove the mod without editing the .love directly in updater.
The same thing will be a problem when trying to bring pacci characters and stages to the players so we'll have to think about this from a design perspective (removing user options to remove default assets, yay or nay?) and work on a general solution. 
For this one I think it will be fine to merge and advertise the pacci panels in #announcements until we have a better thought-out solution that can be applied to all new assets.

**Comment by Endaris** [comment]
@Shosoul  the dumbest but simplest idea I have for delivering the panels at the moment is to simply always copy them if they're not there on startup until a certain date. So they'll be "undeleteable" until that date and then resume normal functionality.
```Lua
if os.time() < os.time({year = 2024, month = 1, day = 31}) and not panels["pacci"] then
    recursive_copy("panels/__default", "panels/pacci")
    add_panels_from_dir_rec("panels/pacci")
  end
```

File: panels.lua [hunk 2]
```diff
@@ -85,7 +90,7 @@ function Panels.load(self)
   local function load_panel_img(name)
     local img = GraphicsUtil.loadImageFromSupportedExtensions(self.path .. "/" .. name)
     if not img then
-      img = GraphicsUtil.loadImageFromSupportedExtensions("panels/__default/" .. name)
+      img = GraphicsUtil.loadImageFromSupportedExtensions("panels/pacci/" .. name)
```
Comment (Shosoul) [inline panels.lua:93]:
```suggestion
      img = GraphicsUtil.loadImageFromSupportedExtensions("panels/__default/" .. name)
```

**Comment by Shosoul** [comment]
> @Shosoul the dumbest but simplest idea I have for delivering the panels at the moment is to simply always copy them if they're not there on startup until a certain date. So they'll be "undeleteable" until that date and then resume normal functionality.
> 
> ```lua
> if os.time() < os.time({year = 2024, month = 1, day = 31}) and not panels["pacci"] then
>     recursive_copy("panels/__default", "panels/pacci")
>     add_panels_from_dir_rec("panels/pacci")
>   end
> ```

I'm not totally against the idea. Maybe we add debug options to the debug menu to re-deploy the default stuff and let people know about it?

**Comment by Endaris** [comment]
Going with the temporary solution for now, we can discuss more permanent measures in the meantime.

**Pull request merged**